### PR TITLE
Make zone and region node labels conformant.

### LINKED ISSUE #17506: Make zone and region node labels consistent with our convention
Fix https://github.com/kubernetes/kubernetes/pull/16866/files#r44672302

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#label-selector-and-annotation-conventions

Should have been something like:
failure-domain.kubernetes.io/zone
failure-domain.kubernetes.io/region

### PR DESCRIPTION
**Pull request by ghost**
Fixes #17506 

@justinsb @bgrant0607 FYI

### CODE REVIEW

File: pkg/api/unversioned/well_known_labels.go [hunk 1]
```diff
@@ -16,5 +16,5 @@ limitations under the License.
 
 package unversioned
 
-const LabelZoneFailureDomain = "kubernetes.io/zone-failure-domain"
-const LabelZoneRegion = "kubernetes.io/zone-region"
+const LabelZoneFailureDomain = "failure-domain.alpha.kubernetes.io/zone"
```
Comment (caesarxuchao) [inline pkg/api/unversioned/well_known_labels.go]: @quinton-hoole out of curiosity, does the "alpha" mean the failure-domain is an alpha feature?

**Comment by caesarxuchao** [comment]
LGTM. Thanks.

File: pkg/api/unversioned/well_known_labels.go (hunk 1 above)
Comment (bgrant0607) [inline pkg/api/unversioned/well_known_labels.go]: Yes

**Comment by bgrant0607** [comment]
Thanks much. LGTM

**Pull request merged**